<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MarkItDown</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuInsrer">
    <property name="title">
     <string>Ins√©rer</string>
    </property>
    <addaction name="actionRule"/>
    <addaction name="actionInlineCode"/>
    <addaction name="actionCodeSnippet"/>
    <addaction name="actionLink"/>
    <addaction name="actionList"/>
    <addaction name="actionUList"/>
    <addaction name="actionQuote"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuInsrer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionStrike"/>
   <addaction name="actionLink"/>
   <addaction name="actionCodeSnippet"/>
   <addaction name="actionInlineCode"/>
   <addaction name="actionUList"/>
   <addaction name="actionList"/>
   <addaction name="actionRule"/>
   <addaction name="separator"/>
   <addaction name="actionQuote"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/file.png</normaloff>:/icon/icons/file.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/floppy-disk.png</normaloff>:/icon/icons/floppy-disk.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/013-cut.png</normaloff>:/icon/icons/013-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/014-paste.png</normaloff>:/icon/icons/014-paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/011-copy.png</normaloff>:/icon/icons/011-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/059-plus sign.png</normaloff>:/icon/icons/059-plus sign.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/058-undo.png</normaloff>:/icon/icons/058-undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/044-redo.png</normaloff>:/icon/icons/044-redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/004-bold.png</normaloff>:/icon/icons/004-bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/031-italic.png</normaloff>:/icon/icons/031-italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionLink">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/033-link.png</normaloff>:/icon/icons/033-link.png</iconset>
   </property>
   <property name="text">
    <string>Insert link</string>
   </property>
  </action>
  <action name="actionList">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/038-number.png</normaloff>:/icon/icons/038-number.png</iconset>
   </property>
   <property name="text">
    <string>Insert Ordered List</string>
   </property>
  </action>
  <action name="actionInlineCode">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/009-code.png</normaloff>:/icon/icons/009-code.png</iconset>
   </property>
   <property name="text">
    <string>Insert inline code</string>
   </property>
   <property name="toolTip">
    <string>Inline Code </string>
   </property>
  </action>
  <action name="actionRule">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/048-delete.png</normaloff>:/icon/icons/048-delete.png</iconset>
   </property>
   <property name="text">
    <string>Insert Rule</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/016-crossed.png</normaloff>:/icon/icons/016-crossed.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionStrike">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/053-strikethrough.png</normaloff>:/icon/icons/053-strikethrough.png</iconset>
   </property>
   <property name="text">
    <string>Strike</string>
   </property>
  </action>
  <action name="actionUList">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/002-alphabet.png</normaloff>:/icon/icons/002-alphabet.png</iconset>
   </property>
   <property name="text">
    <string>Insert Unordered List</string>
   </property>
  </action>
  <action name="actionQuote">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/056-quote.png</normaloff>:/icon/icons/056-quote.png</iconset>
   </property>
   <property name="text">
    <string>Insert Quote</string>
   </property>
  </action>
  <action name="actionCodeSnippet">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/icon/icons/programing.png</normaloff>:/icon/icons/programing.png</iconset>
   </property>
   <property name="text">
    <string>Insert code snippet</string>
   </property>
   <property name="toolTip">
    <string>Code Snippet</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MyRes.qrc"/>
 </resources>
 <connections/>
</ui>
